lib/utils/constants.dart

```diff
@@ -0,0 +1,11 @@
+abstract class Constants {
+  static const int crossAxisCountNumber = 2;
+  static const double paddingNumber = 10.0;
+  static const String imageNetwork =
+      "https://image.tmdb.org/t/p/w185";
+
+  static const String uriClient =
+      "https://api.themoviedb.org/3/movie/popular?api_key=385737c6d715026a1cd89526ff9d8d44";
+  static const int status = 200;
+  static const String error = 'I failed the load';
```

Comment:
This error message can be: 'An error has ocurred while loading movies"

Reply:
done